haystack/finder.py

```diff
@@ -17,15 +17,15 @@ def __init__(self, reader, retriever):
         self.retriever = retriever
         self.reader = reader
 
-    def get_answers(self, question, top_k_reader=1, top_k_retriever=10, filters=None):
+    def get_answers(self, question: str, top_k_reader: int = 1, top_k_retriever: int = 10, filters: dict = None):
         """
         Get top k answers for a given question.
 
         :param question: the question string
         :param top_k_reader: number of answers returned by the reader
         :param top_k_retriever: number of text units to be retrieved
         :param filters: limit scope to documents having the given tags and their corresponding values.
-            The format for the dict is {"tag-1": ["value-1","value-2"], "tag-2": ["value-3]" ...}
+            The format for the dict is {"tag-1": "value-1", "tag-2": "value-2" ...}
```

Comment:
Is this update really correct? I updated the docstring to {"tag-1": ["value-1","value-2"], "tag-2": ["value-3]" ...} in https://github.com/deepset-ai/haystack/commit/0cffc6cb1df8782bbf1bbc336807de355687584b#diff-ba671fc4346d75ce42ce99420a310389L28